.github/workflows/video-update.yml

```diff
@@ -0,0 +1,23 @@
+name: Video
+on:
+  push:
+    branches:
+      - master
+jobs:
+  up:
+    name: Update
```

Comment:
Same here. Perhaps this should also be fixed to only run in this repo and not forks?